src/app/components/selected-country-data/selected-country-data.component.ts

```diff
@@ -68,7 +75,7 @@ export class SelectedCountryDataComponent implements OnInit {
   /** Back to previous screen */
   back(): void {
     const regParam = sessionStorage.getItem('region');
-    if (regParam !== 'undefined') {
+    if (regParam !== 'undefined') { // if region is not present in the sessionStorage it will return null, not a string
```

Comment:
but getting undefined in the console

Reply:
that's because you're setting it to undefined somewhere

Reply:
ok will check that